### ISSUE TITLE
Request payload is different from one sent, due to json parsing

### ISSUE THREAD

**Issue by CRogers** [probot/smee.io#78]
When you hit smee with some json data, for example:

```json
{
   "a": 2
}
```

`smee` parses the json and then reformats it when you get it out of the request:

```json
data: {"host":"my-local-smee",...,"body":{"a":2},...}
```

This is unfortunate as you don't get the exact same data back. In my case, I'm also getting a shared-secret HMAC signature along with the payload, which no longer matches the expected signature. This means that I can't test my payload signature checking code is working with the external service and have to disable it, which is sad.

**Comment by CRogers** [comment]
I guess #5 is also an effect of this (content-length changes due to the payload being changed).

**Comment by mosheco** [comment]
When can a new Smee release , fixing this issue, be expected ?

**Comment by wolfy1339** [comment]
I don't have control of the infrastructure running the website code. I merged #55 which allows you to have the raw body of a request
